src/partition/repart.c

```diff
@@ -6795,6 +6850,43 @@ static int parse_argv(int argc, char *argv[]) {
                 return log_error_errno(SYNTHETIC_ERRNO(EINVAL),
                                        "Expected at most one argument, the path to the block device.");
 
+        if (arg_make_ddi) {
+                if (arg_definitions)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Combination of --make-ddi= and --definitions= is not supported.");
+                if (arg_factory_reset > 0)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Combination of --make-ddi= and --factory-reset= is not supported.");
+                if (arg_size != UINT64_MAX)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Combination of --make-ddi= and --size= is not supported.");
+                if (arg_image || arg_root)
+                        return log_error_errno(SYNTHETIC_ERRNO(EINVAL), "Combination of --make-ddi= and --image=/--root= is not supported.");
+
+                if (!arg_copy_source) {
+                        r = safe_getcwd(&arg_copy_source);
+                        if (r < 0)
+                                return log_error_errno(r, "Failed to determine current working directory: %m");
+                }
```

Comment:
I think this will be more confusing than useful. It's really not that much effort to add `-s .` or `--root .` to the CLI invocation

Reply:
ok, will drop.